src/atomicdex/config/addressbook.cfg.cpp

```diff
@@ -0,0 +1,71 @@
+/******************************************************************************
+ * Copyright © 2013-2019 The Komodo Platform Developers.                      *
+ *                                                                            *
+ * See the AUTHORS, DEVELOPER-AGREEMENT and LICENSE files at                  *
+ * the top-level directory of this distribution for the individual copyright  *
+ * holder information and the developer policies on copyright and licensing.  *
+ *                                                                            *
+ * Unless otherwise agreed in a custom licensing agreement, no part of the    *
+ * Komodo Platform software, including this file may be copied, modified,     *
+ * propagated or distributed except according to the terms contained in the   *
+ * LICENSE file                                                               *
+ *                                                                            *
+ * Removal or modification of this copyright notice is prohibited.            *
+ *                                                                            *
+ ******************************************************************************/
+ 
+//! STD
+#include <fstream> //> std::fstream, std::ifstream, std::ofstream.
+
+//! Project Headers
+#include "addressbook.cfg.hpp"
+#include "atomicdex/utilities/global.utilities.hpp"
+
+namespace atomic_dex
+{
+    nlohmann::json load_addressbook_cfg(const std::string& wallet_name)
+    {
+        fs::path       source_folder{utils::get_atomic_dex_addressbook_folder()};
+        fs::path       in_path      {source_folder / wallet_name};
+        std::fstream   input;
+        nlohmann::json out;
+        
+        utils::create_if_doesnt_exist(source_folder);
+        {
+            //! Creates file if it does not exist.
+            if (!std::ifstream(in_path.string()))
+            {
+                input.open(in_path.string(), std::ios::out);
+                assert(input.is_open());
```

Comment:
i believe `std::fstream` have a boolean operator that doesn't require to call `is_open`